src/prism-agent/Agent.ts

```diff
@@ -1,19 +1,12 @@
-import { Apollo } from "../domain/buildingBlocks/Apollo";
-import {
-  DID,
-  Message,
-  Seed,
-  Service as DIDDocumentService,
-  Signature,
-  Credential,
-} from "../domain";
-import { Castor } from "../domain/buildingBlocks/Castor";
-import { Pluto } from "../domain/buildingBlocks/Pluto";
-import { Mercury } from "../domain/buildingBlocks/Mercury";
-import { Api } from "../domain/models/Api";
+import * as Domain from "../domain";
+
+import Apollo from "../apollo";
+import Castor from "../castor";
+import Mercury from "../mercury";
+import Pollux from "../pollux";
```

Comment:
Im not a TS developer but this will make the Agent directly dependent on this implementations of `Apollo`, `Castor`, `Mercury` and `Pollux`?

But in other way I see what you did here. You basically made the Agent module dependent on the Building blocks modules, but the agent API is still dependent on the `Domain.Apollo`, etc.

So you can create a initialisation that uses the our implementations as defaults. Cool, I would have done it in a different way since I prefer the Agent to not ever know of the BBs module. But I really see the value of this its less confusing and it has quite some value.